applications/dashboard/models/class.usermodel.php

```diff
@@ -941,7 +941,6 @@ public function joinUsers(&$Data, $Columns, $Options = array()) {
 
         // Join the user data using prefixes (ex: 'Name' for 'InsertUserID' becomes 'InsertName')
         $Join = val('Join', $Options, array('Name', 'Email', 'Photo'));
-        $UserPhotoDefaultUrl = function_exists('UserPhotoDefaultUrl');
```

Comment:
With this removed, activity items like "x users joined" are missing the profile picture if the first user has only a vanillicon. (ActivityModel::getWhere gets the avatars from UserModel::joinUsers)